modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm

```diff
@@ -66,7 +66,6 @@ sub run {
     my $self = shift @_;
 
     my $all_trees = $self->param('current_tree_adaptor')->fetch_all( -TREE_TYPE => 'tree', -MEMBER_TYPE => 'protein', -CLUSTERSET_ID => 'default' );
-
```

Comment:
Is it necessary to delete this blank line? Making a whitespace change like this makes it more difficult to focus on the relevant changes.

Reply:
Sorry, it is a residual change due to the tests I've performed on the code. However, the extra blank line makes the code less condensed to read.

Reply:
@thiagogenez, can you please revert this to how it is on the `release/111` branch ([here](https://github.com/Ensembl/ensembl-compara/blob/0858b045bccd870f8b44694df14ec93ef84f8cb6/modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm#L69))? You have restored the blank line, but now it contains whitespace that wasn't present before, so this is still an extraneous whitespace change.